src/main/java/com/openlattice/chronicle/sources/AndroidDevice.java

```diff
@@ -1,40 +1,34 @@
 package com.openlattice.chronicle.sources;
 
-import static com.openlattice.chronicle.ChronicleFields.ADDITIONAL_INFO;
-import static com.openlattice.chronicle.ChronicleFields.BRAND;
-import static com.openlattice.chronicle.ChronicleFields.CODENAME;
-import static com.openlattice.chronicle.ChronicleFields.DEVICE;
-import static com.openlattice.chronicle.ChronicleFields.DEVICE_ID;
-import static com.openlattice.chronicle.ChronicleFields.MODEL;
-import static com.openlattice.chronicle.ChronicleFields.OS_VERSION;
-import static com.openlattice.chronicle.ChronicleFields.PRODUCT;
-import static com.openlattice.chronicle.ChronicleFields.SDK_VERSION;
-
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.google.common.base.Optional;
+
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
 
+import static com.openlattice.chronicle.ChronicleFields.*;
```

Comment:
I generally don't agree with static imports because I think it takes away from readability at best and at worse imports more than is used, leading to subtle confusing bugs. 
This is a pretty constrained case but I want to flag the `import static *` generally